requirements.txt

```diff
@@ -1,2 +1,2 @@
-sigstore ~= 3.0
+sigstore ~= 3.4
```

Comment:
Is there a reason to not just pin the version exactly?

Reply:
No good reason -- if you prefer an exact pin I can move to that!